Assets/Scripts/Content.cs

```diff
@@ -3,13 +3,14 @@
 using UnityEngine;
 using TMPro;
 using UnityEngine.SceneManagement;
+using System.Security.Cryptography.X509Certificates;
 
 public class Content : MonoBehaviour
 {
     public TextMeshProUGUI m_textComponent; //reference tmpro to text component
     const int TALKERNUM = 2;
     const int LINESNUM = 9;
-    private string[] m_lines=new string[LINESNUM];
+    private string[] m_lines = new string[LINESNUM];
```

Comment:
Just remember in any place with m_ or s_ the next word needs to be capital so in this case m_Lines